content/en/quickstart/quickstart-cosign.md

```diff
@@ -1,22 +1,26 @@
 ---
 type: docs
-category: Signing
-description: ''
-menuTitle: Quick Start
-title: Cosign
-weight: 100
+category: Quickstart
+description: Sign and Verify with Cosign
+title: Sigstore Quickstart with Cosign
+weight: 5
 ---
 
 ![Cosign Overview](/sigstore_cosign-horizontal-color.svg)
 
-## Getting Started (Quick Start)
-
-Cosign is a command line utility that can sign and verify software artifact, such as container images and blobs.
 Join us on our [Slack channel](https://sigstore.slack.com/). (Need an [invite](https://links.sigstore.dev/slack-invite)?)
 
+## Quickstart signing and verifying with Cosign
+
+Cosign is a command line utility that is used to sign software artifacts and verify signatures using Sigstore.
+
+Language specific clients (like [sigstore-python](https://github.com/sigstore/sigstore-python)) are other options for signing and verifying, but Cosign is a great, language agnostic place to start. 
```

Comment:
I might slightly rephrase to add that the language specific clients are SDKs to build tooling on top of, and though some may offer basic CLIs, Cosign is the recommended tool for most signing and verifying.

Reply:
@haydentherapper Thanks! That makes sense. I have updated the wording a bit. This part will change again once the language clients are added to the docs. We could also cut this section for now and reintroduce something once there is something more to link to within the documentation (I want to link to the upcoming 'language client' section of the docs).